src/Entity/Esin.php

```diff
@@ -201,11 +201,15 @@ public function setEmissionDeLaFiche(?\DateTimeInterface $emissionDeLaFiche): se
 
     public function setEmissionString(string $date)
     {
-        $day = substr($date, 0, 2);
-        $month = substr($date, 3, 2);
-        $year = substr($date, 6, 4);
-        $this->emissionDeLaFiche = new DateTime($year ."-". $month . "-". $day);
+        /*
+        $dateString = str_replace("/", "-", $date);
+        try {
+            $this->emissionDeLaFiche = new DateTime($dateString);
+        } catch (Exception $e) {
+            throw new Exception("Le format de la date est erroné");
+        }
         return $this;
+        */
```

Comment:
You should keep dead code (code that is never executed, commented one for instance) in your repository